cmsdb/processes/higgs.py

```diff
@@ -203,6 +246,11 @@
             "pdf": 0.03j,
             "mtop": (0.04j, 0.18j),
         }),
+        13.6: Number(0.03443, {
+            "scale": (0.021j, 0.049j),
+            "pdf": 0.03j,
+            "mtop": (0.04j, 0.18j),
+        }),
```

Comment:
```suggestion
        13.6: Number(0.03443, {  # value for mH=125 GeV
            "scale": (0.021j, 0.049j),
            "pdf": 0.03j,
            "mtop": (0.04j, 0.18j),
        }),
```



I would add a comment to say that the values are for mH=125GeV (for bookkeeping)